kibana-reports/public/components/report_definitions/edit/edit_report_definition.tsx

```diff
@@ -200,6 +239,9 @@ export function EditReportDefinition(props) {
           'error when loading edit report definition page: ',
           error
         );
+        if (error.body.statusCode === 403) {
+          handlePermissionsMissingViewEditPageToast();
+        }
```

Comment:
missing `else`?

Reply:
Good catch, we can add an `API Error` toast for loading report definition values